chart/templates/99-ingress-nginx.yml

```diff
@@ -9,16 +9,25 @@ metadata:
     nginx.ingress.kubernetes.io/preserve-trailing-slash: "false"
     nginx.ingress.kubernetes.io/ssl-passthrough: "false"
     nginx.ingress.kubernetes.io/client-body-buffer-size: "512k"
+{{- if .Values.ingress.proxyBufferBypass }}
```

Comment:
allow user to include or exclude this proxy buffer bypass stuff (it seems that possibly different ingress-nginx versions behave a bit differently, I may still dig into this more). default behavior is that this is `true` meaning the same behavior as before